docs/development/process.md

```diff
@@ -27,7 +27,12 @@ Hotfixes are usually only maintained for the latest minor release as well as the
 
 The validation process for new releases usually takes a couple of days and includes the following steps:
 
-1. `master` (or latest `release-*` branch) is deployed to a development landscape that already hosts some existing seed and shoot clusters.
+1. If a new
+   1. minor release `vX.Y.0` shall be cut then a new branch `release-vX.Y` is created based on `master`. [*]
+   1. patch release `vX.Y.Z` shall be cut then depending on `Z`:
+      1. If `Z=1` then a new branch `release-vX.Y` is created based on the `vX.Y.0` tag.
```

Comment:
If we create `release-vX.Y` already in the step above for the minor release, creating it won't be necessary for the patch release.

Reply:
Of course, thanks :)